### PR TITLE
Add Lodash link

### LINKED ISSUE #87: Add link to Lodash Website in README
We want to add a link to the highlighted text below that redirects to the Lodash official documentation.

Screenshot_20191012-153950_Chrome

### PR DESCRIPTION
**Pull request by daningenthron**
Fixes #87 - adds Lodash link to CONTRIBUTING.md. First time contribution - this is a very cool library! Happy to add more as needed.

### CODE REVIEW

**Comment by Attrash-Islam** [comment]
Thanks @daningenthron! Appreciate the PR and would be glad to see more ❤🌷

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ ruby ./test/reduce.rb
 
 Some notes to be aware of:
 
-- You **MUST** be aligned to the Lodash behavior as much as possible.
+- You **MUST** be aligned to the Lodash behavior as much as possible. (I try cases in the console of the [Lodash website](https://lodash.com).)
```
Comment (Attrash-Islam) [inline CONTRIBUTING.md]: Please update the link to point to -> https://lodash.com/docs/4.17.15

Comment (Attrash-Islam) [reply CONTRIBUTING.md]: Also, I thought of dropping this (I try cases ...) with adding the link to the "Lodash behavior" text. WDYT @daningenthron?

**Comment by Attrash-Islam** [comment]
Thanks!! That was so fast. Merging 🎉

File: CONTRIBUTING.md (hunk 1 above)
Comment (daningenthron) [reply CONTRIBUTING.md]: I like it! Less wordy and more specific. I've updated the commit and bumped the version too.

**Pull request merged**